examples/atlas-api-key/README.md

```diff
@@ -0,0 +1,16 @@
+# MongoDB Atlas Provider -- MongoDB Atlas Programmatic API Key Examples 
+
+In the Terraform MongoDB Atlas Provider v1.8.0 release, we introduced support for MongoDB Atlas Programmatic API Keys (PAK). While this was an exciting development, we quickly realized that some of our customers were facing challenges in optimally leveraging this resource. In response to this, we initiated several enhancements as part of the v1.10.0 release to refine the user experience with this resource. These enhancements encompassed both code revisions and documentation updates.
+
+The most notable improvements among these as part of v1.10.0 release, was the deprecation of the `api_keys` parameter from the `mongodbatlas_project` resource. We also extended the functionality of the `mongodbatlas_project_api_key` resource by incorporating the `project_assignment` parameter. This enhancement removes the necessity for users to create multiple `mongodbatlas_project` resource blocks just to assign keys. This update streamlines the process of assigning an API Key to multiple projects, making it less cumbersome and more manageable.
```

Comment:
suggestion, feel free to take or leave - I think this first sentence could be simplified to something like:
"The most notable enhancement among these was the deprecation of the `api_keys` parameter from the ..."

otherwise, just a tweak to what's there:
```suggestion
The most notable improvement among these as part of v1.10.0 release was the deprecation of the `api_keys` parameter from the `mongodbatlas_project` resource. We also extended the functionality of the `mongodbatlas_project_api_key` resource by incorporating the `project_assignment` parameter. This enhancement removes the necessity for users to create multiple `mongodbatlas_project` resource blocks just to assign keys. This update streamlines the process of assigning an API Key to multiple projects, making it less cumbersome and more manageable.
```